marshal/oracle/listpdb_test.go

```diff
@@ -0,0 +1,36 @@
+package oracle
+
+import (
+	"github.com/ercole-io/ercole/v2/model"
+	"github.com/stretchr/testify/assert"
```

Comment:
The same as in `listpdb.go`!
Imports can't be mixed between standard libraries and other sources